### DISCUSSION TITLE
Orbit and Filament

### DISCUSSION THREAD

**Discussion by NigelDorning** [ryangjchandler/orbit#49] [General]
Hi,

Interested in playing around with this and Filament and just wondering if there is a way to store Filament users as flat files?

I've tried extending the Filament User model and adding the Orbital trait and static schema method but that didn't seem to work.

Sorry if this isn't totally specific to this package by the way. I can move it if necessary.

Thanks a lot.

**Comment by ryangjchandler** [comment]
Just extending the model won't work because it's not setup with Filament's guard. 

You can read Filament's docs on using a custom model & auth guard, then follow https://ryangjchandler.co.uk/articles/storing-laravel-users-on-disk-in-flat-files to learn how to create a flat User model.

**Reply by NigelDorning** [reply]
Excellent. I'll take a look at that.

Thanks for the reply.

**Comment by rasul-chopurov** [comment]
`namespace App\Models;

use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Schema\Blueprint;

class Post extends Model
{
    use \Orbit\Concerns\Orbital;

    public static $driver = 'json';
  

    public static function schema(Blueprint $table)
    {
        $table->string('title');
        $table->string('slug');
    }
}`

not work Add [title] to fillable property to allow mass assignment on [App\Models\Post].

**Reply by ryangjchandler** [reply]
Orbit models are regular Eloquent models, so regular mass assignment rules apply. 

You must either create a `$fillable` property on the model with all of the mass-assignable attributes, or set `$guarded = []`.

**Reply by rasul-chopurov** [reply]
not work in filament resources use orbit :/ 
filiable or guarded ..

**Reply by ryangjchandler** [reply]
Please share the new error after adding `fillable` or `guarded`.
